reward/util.py

```diff
@@ -509,18 +510,36 @@ def calc_strain_energy(docking_pose_file, conf):
     return total_strain_energy, max_single_strain_energy
 
 
-# This function currently supports GNINA.
 def get_interaction_distances(receptor_fname, output_ligand_fname, pose_idx, conf):
 
     # Import necessary modules
     plf = sys.modules.get('prolif') or __import__('prolif')
     mda = sys.modules.get('MDAnalysis') or __import__('MDAnalysis')
 
-    # Load receptor and docked ligand
-    protein_file = receptor_fname.replace('/scr', 'data')
-    u = mda.Universe(protein_file)
+    # Load receptor
+    if receptor_fname.startswith('/scr'):
```

Comment:
Please add a comment: "This process is for cases where GNINA reward is used".

Reply:
I added the comment. (b3584684ede807fea7024d270e448f69981378ef)